### PR TITLE
US001 React Ts Template 

### PR DESCRIPTION
**Pull request by Zeroya**
Created project with npx create-react-app . --template redux-typescript, cleaned up the files and changed the location of the folders, deployed to github pages.

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -1,43 +1,21 @@
...
+  <title>React Redux App</title>
+</head>
+
+<body>
+  <noscript>You need to enable JavaScript to run this app.</noscript>
+  <div id="root"></div>
+
+</body>
+
+</html>
```
Comment (Yuliia143) [inline public/index.html]: GitHub needs empty line at the end of each file you want to add

File: src/hooks/hooks.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+import { useDispatch, useSelector } from 'react-redux'
+import type { TypedUseSelectorHook } from 'react-redux'
+import type { RootState, AppDispatch } from '../store/store'
+
+export const useAppDispatch: () => AppDispatch = useDispatch
+export const useAppSelector: TypedUseSelectorHook<RootState> = useSelector
```
Comment (Yuliia143) [inline src/hooks/hooks.ts]: GitHub needs empty line at the end of each file you want to add

File: src/store/reducers/UserSlice.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+  reducers: {
+    increment: (state) => {
+      state.value += 1
+    },
+  },
+})
+
+export const { increment } = counterSlice.actions
+
+export default counterSlice.reducer
```
Comment (Yuliia143) [inline src/store/reducers/UserSlice.ts]: GitHub needs empty line at the end of each file you want to add

File: src/store/store.ts [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+export const store = configureStore({
+  reducer: rootReducer
+    
+})
+
+
+export type RootState = ReturnType<typeof store.getState>
+
+export type AppDispatch = typeof store.dispatch
```
Comment (Yuliia143) [inline src/store/store.ts]: GitHub needs empty line at the end of each file you want to add

File: src/hooks/hooks.ts [hunk 5]
```diff
@@ -0,0 +1,6 @@
+import { useDispatch, useSelector } from 'react-redux'
```
Comment (Yuliia143) [inline src/hooks/hooks.ts]: Follow one code style, configure eslint to add semicolon or not

File: src/App.tsx [hunk 6]
```diff
@@ -1,56 +1,17 @@
 import React from 'react';
-import logo from './logo.svg';
-import { Counter } from './features/counter/Counter';
+import { useAppDispatch, useAppSelector } from './hooks/hooks';
```
Comment (Yuliia143) [inline src/App.tsx:2]: Follow one code style, configure eslint to add semicolon or not

File: src/store/reducers/UserSlice.ts [hunk 7]
```diff
@@ -0,0 +1,29 @@
+import { createSlice } from '@reduxjs/toolkit'
```
Comment (Yuliia143) [inline src/store/reducers/UserSlice.ts]: Follow one code style, configure eslint to add semicolon or not

File: src/store/store.ts [hunk 8]
```diff
@@ -0,0 +1,18 @@
+import { combineReducers, configureStore } from '@reduxjs/toolkit'
+import userReducer from './reducers/UserSlice'
+// ...
```
Comment (Yuliia143) [inline src/store/store.ts]: Do you need this line ?

File: src/store/store.ts [hunk 9]
```diff
@@ -0,0 +1,18 @@
...
+
+const rootReducer = combineReducers({
+  value : userReducer
+})
+
+
+export const store = configureStore({
+  reducer: rootReducer
+
```
Comment (Yuliia143) [inline src/store/store.ts]: Follow the same indents

**Review by Yuliia143 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (Zeroya) [reply public/index.html]: Сhanges added

File: src/hooks/hooks.ts (hunk 2 above)
Comment (Zeroya) [reply src/hooks/hooks.ts]: Сhanges added

File: src/store/reducers/UserSlice.ts (hunk 3 above)
Comment (Zeroya) [reply src/store/reducers/UserSlice.ts]: Сhanges added

File: src/store/store.ts (hunk 4 above)
Comment (Zeroya) [reply src/store/store.ts]: Сhanges added

File: src/hooks/hooks.ts (hunk 5 above)
Comment (Zeroya) [reply src/hooks/hooks.ts]: Сhanges added

File: src/App.tsx (hunk 6 above)
Comment (Zeroya) [reply src/App.tsx:2]: Сhanges added

File: src/store/reducers/UserSlice.ts (hunk 7 above)
Comment (Zeroya) [reply src/store/reducers/UserSlice.ts]: Сhanges added

File: src/store/store.ts (hunk 8 above)
Comment (Zeroya) [reply src/store/store.ts]: No. Сhanges added

File: src/store/store.ts (hunk 9 above)
Comment (Zeroya) [reply src/store/store.ts]: I will. Сhanges added

**Review by Yuliia143 — APPROVED** [review]

**Pull request merged**